torch/distributed/fsdp/_init_utils.py

```diff
@@ -371,7 +371,7 @@ def _init_device_handle(
     If a device is specified by ``device_id``,
     then returns device handle corresponds to that device type. Otherwise, If the
     module is already on a non-CPU device, then the device type is that non-CPU device type.
-    If the module is on CPU or meta, then the device type is the current cuda device.
+    If the module is on CPU or meta, then the device type is the current device.
```

Comment:
"current accelerator device". And we can link to the _accelerators pin above.

Reply:
I have added it.